examples/file-and-cloudinary-upload/app/routes/cloudinary-upload.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+  Form,
+  unstable_createFileUploadHandler,
+  unstable_parseMultipartFormData,
+  useActionData,
+  json
+} from "remix";
+import type { ActionFunction } from "remix";
+import { cloudinary } from "~/utils/utils.server";
+
+type ActionData = {
+  errorMsg?: string;
+  imgSrc?: string;
+};
+
+export const action: ActionFunction = async ({ request }) => {
+  const uploadHandler = unstable_createFileUploadHandler({
+    maxFileSize: 30000
+  });
```

Comment:
Rather that saving the image to disk first and then uploading it like we're doing here, we should try to get streaming working (as demonstrated here: https://remix.run/docs/en/v1/api/remix#custom-uploadhandler).

Could you try to get that working so we can demonstrate that instead? It's more optimized and realistic.

Reply:
sure will give a try.